### PR TITLE
feat(patterns/radio): update radio pattern to use it in forms

### PR DESCRIPTION
**Pull request by caqueste**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

**Comment by caqueste** [comment]
Deployment link → https://featid676formradio-dot-demo-dot-mozaic-design-system-dtrp.ey.r.appspot.com

File: src/docs/Components/Form/Radio/index.mdx [hunk 1]
```diff
@@ -11,6 +11,67 @@ status:
```
Comment (nherwegh) [inline src/docs/Components/Form/Radio/index.mdx:11]: The "definition" is very weird to me. It doesn't define what it is or what it does but it's being opposed to checkboxes. Can we try something like: "A radio button is used to offer a unique choice to your user in a form."

File: src/docs/Components/Form/Radio/index.mdx [hunk 2]
```diff
@@ -11,6 +11,67 @@ status:
...
+
+<Highlight theme="warning">
+
+  A radio button can't be used alone, it must be used within a group.
+
+</Highlight>
+
+![Radio-Overview](Radio-Overview.png)
+
+Checkbox group is a combination of the following mentions. Please note that you can use four different cases:
```
Comment (nherwegh) [inline src/docs/Components/Form/Radio/index.mdx:25]: Radio button not "checkbox" 😉

Comment (mohamedMok) [inline src/docs/Components/Form/Radio/index.mdx:25]: Radio?

File: src/docs/Components/Form/Radio/index.mdx [hunk 3]
```diff
@@ -11,6 +11,67 @@ status:
...
+
+There are two different elements for validation feedback:
+  - The color of the input outline.
+  - A text under the radio button group explains what's wrong with the data entered.
+
+<Preview path="group-invalid-light" nude />
+
+
+
 ## Do & dont's
```
Comment (nherwegh) [inline src/docs/Components/Form/Radio/index.mdx:75]: A radio buttons list must include a minimum of two options.

File: src/docs/Components/Form/Radio/index.mdx [hunk 4]
```diff
@@ -11,6 +11,67 @@ status:
...
```
Comment (nherwegh) [inline src/docs/Components/Form/Radio/index.mdx:76]: Never use the radio alone. You should always use it with a label and within a group.

File: src/docs/Components/Form/Radio/index.mdx (hunk 2 above)
Comment (caqueste) [reply src/docs/Components/Form/Radio/index.mdx:25]: Fixed ;)

Comment (caqueste) [reply src/docs/Components/Form/Radio/index.mdx:25]: Fixed ;)

File: src/docs/Components/Form/Radio/index.mdx (hunk 4 above)
Comment (caqueste) [reply src/docs/Components/Form/Radio/index.mdx:76]: Fixed

File: src/docs/Components/Form/Radio/index.mdx (hunk 3 above)
Comment (caqueste) [reply src/docs/Components/Form/Radio/index.mdx:75]: Fixed

File: src/docs/Components/Form/Radio/index.mdx [hunk 5]
```diff
@@ -1,26 +1,89 @@
...
 order: 3
-description: 'Radios are components that, unlike checkboxes, can’t be used alone. They are used to select a single option from a list.'
+description: 'A radio button is used to offer a unique choice to your user in a form. Unlike checkboxes, it can not be used alone.'
 status:
   sketch: 'bêta'
   scss: 'bêta'
 ---
 
-> Radios are components that, unlike checkboxes, can't be used alone. They are used to select **a single option from a list**.
+> A radio button is used to offer a unique choice to your user in a form. Unlike checkboxes, it can't be used alone.
```
Comment (nherwegh) [inline src/docs/Components/Form/Radio/index.mdx:10]: change can't to can not also here 😉

**Pull request merged**